static/app/views/releases/list/releasesPromo.tsx

```diff
@@ -79,81 +86,356 @@ export const RELEASES_TOUR_STEPS: TourStep[] = [
-          </Button>
-          <FeatureTourModal
-            steps={RELEASES_TOUR_STEPS}
-            onAdvance={this.handleTourAdvance}
-            onCloseModal={this.handleClose}
-            doneText={t('Start Setup')}
-            doneUrl={releasesSetupUrl}
-          >
-            {({showModal}) => (
-              <Button priority="default" onClick={showModal}>
-                {t('Take a Tour')}
-              </Button>
-            )}
-          </FeatureTourModal>
-        </ButtonList>
-      </OnboardingPanel>
+  const generateToken = async (sentryAppSlug: string) => {
+    const newToken = await api.requestPromise(
+      `/sentry-apps/${sentryAppSlug}/api-tokens/`,
+      {
+        method: 'POST',
+      }
     );
+    return newToken.token;
+  };
+
+  const renderIntegrationNode = (integration: SentryApp) => {
+    return {
+      value: {slug: integration.slug, name: integration.name},
+      searchKey: `${integration.name}`,
+      label: (
+        <MenuItemWrapper
+          data-test-id="integration-option"
+          key={integration.uuid}
+          // onSelect={handleSelect}
+        >
+          <Label>{integration.name}</Label>
+        </MenuItemWrapper>
+      ),
+    };
+  };
+  return renderComponent(
+    <Fragment>
+      <Panel>
+        <Container>
+          <p>
+            {t(
+              'Configuring releases associates new issues with the right version of your code and ensures the right team members are notified of these issues.'
+            )}
+          </p>
+          <StyledPageHeader>
+            <h3>{t('Configure Releases with the CLI')}</h3>
+
+            <Button priority="primary" size="small" href={releasesSetupUrl} external>
+              {t('Full Documentation')}
+            </Button>
+          </StyledPageHeader>
+          <p>
+            Add the following commands to your CI config when you deploy your application.
+          </p>
+
+          <CodeBlock>
+            <CopyButton
+              onClick={async () => {
+                if (!token || !selectedItem) {
+                  addErrorMessage(t('Select an integration for your auth token!'));
+                  return;
+                }
+                const current_text = `
+                # Install the cli
+                curl -sL https://sentry.io/get-cli/ | SENTRY_CLI_VERSION="2.2.0" bash
+                # Setup configuration values
+                SENTRY_AUTH_TOKEN=${token} # From internal integration: ${selectedItem.value.name}
+                SENTRY_ORG=${organization.slug}
+                SENTRY_PROJECT=${project.slug}
+                VERSION=\`sentry-cli releases propose-version\`
+                # Workflow to create releases
+                sentry-cli releases new "$VERSION"
+                sentry-cli releases set-commits "$VERSION" --auto
+                sentry-cli releases finalize "$VERSION"
+                `;
+                await navigator.clipboard.writeText(current_text);
```

Comment:
i think we're still using `copy-text-to-clipboard` for clipboard except for a few areas that needed to use navigator directly

Reply:
probably could since we only target modern browsers 🤷 i think there's a small chance of hitting a permissions issue that the old way of copying doesn't have